### ISSUE TITLE
Error When using 4 letters Tokens

### ISSUE THREAD

**Issue by maxadss** [jesse-ai/jesse#100]
tried using DOGE/BTC as sample 
Returns "Uncaught Exception: KeyError: 'EBTC' "
bot will store 2 symbols DOG,EBTC and will exit with error OR if u add new asset named EBTC it will run Sim with no transactions  
i think the issue from here 

> jesse / helper.py 

```
def base_asset(symbol: str):
    if symbol.endswith('USDT'):
        return symbol[0:len(symbol) - 4]
    if symbol.endswith('USD'):
        return symbol[0:len(symbol) - 3]
    return symbol[0:3]
```

**Comment by lightyear15** [comment]
Check out my PR #98

**Comment by saleh-mir** [comment]
There is an [on-going branch](https://github.com/jesse-ai/jesse/tree/base-quote) that is supposed to completely change how we define symbols (from `BTCUSD` to `BTC-USD`) that would solve this. I don't have a release date in mind for it, but it'll fix this.

**Comment by saleh-mir** [comment]
This should be fixed with `v0.18.0`
